python/cuspatial/cuspatial/core/binpreds/feature_overlaps.py

```diff
@@ -39,6 +42,15 @@ class OverlapsPredicateBase(EqualsPredicateBase):
     pass
 
 
+class LineStringLineStringOverlaps(BinPred):
+    def _preprocess(self, lhs, rhs):
```

Comment:
A small docstring helps:

```suggestion
    def _preprocess(self, lhs, rhs):
        """Linestring overlaps with another linestring if the overlapping part of 
        two linestring is not the entire section of the linestring.
        """
```
Sounds correct?